src/client/Microsoft.Identity.Client/Instance/Discovery/InstanceDiscoveryMetadataEntry.cs

```diff
@@ -1,21 +1,49 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Linq;
+using Microsoft.Identity.Client.Utils;
 using Microsoft.Identity.Json;
+using Microsoft.Identity.Json.Linq;
 
 namespace Microsoft.Identity.Client.Instance.Discovery
 {
     [JsonObject]
     [Preserve(AllMembers = true)]
-    internal sealed class InstanceDiscoveryMetadataEntry
+    internal sealed class InstanceDiscoveryMetadataEntry : IJsonSerializable<InstanceDiscoveryMetadataEntry>
     {
-        [JsonProperty(PropertyName = "preferred_network")]
+        private const string PreferredNetworkPropertyName = "preferred_network";
+        private const string PreferredCachePropertyName = "preferred_cache";
+        private const string AliasesPropertyName = "aliases";
+
+        [JsonProperty(PropertyName = PreferredNetworkPropertyName)]
```

Comment:
We probably don't need these properties anymore?

Reply:
Left them for now. I have unit tests that compare new serialization behavior with the previous one that use these.